tests/components/plugwise/test_sensor.py

```diff
@@ -158,13 +160,17 @@ async def test_p1_3ph_dsmr_sensor_entities(
     state = hass.states.get(entity_id)
     assert not state
 
-    entity_registry.async_update_entity(entity_id=entity_id, disabled_by=None)
-    await hass.async_block_till_done()
-
-    await hass.config_entries.async_reload(init_integration.entry_id)
-    await hass.async_block_till_done()
 
+@pytest.mark.usefixtures("entity_registry_enabled_by_default")
+async def test_p1_3ph_dsmr_sensor_entities_disabled(
```

Comment:
Not sure if we should make this a separate test?

Reply:
I figured we also test that it's disabled - hence the separate test leveraging the declarator you suggested. Unless it's acceptable that we don't test for entities being disabled?

Reply:
This one is not changed yet in the recent commit, please indicate how we can follow up

Reply:
We don't really test that it's disabled, we only enable them to test them here

Reply:
As discussed; adjusted the route. For clarification - our (@bouwe/me) initial idea was to test some sensor values, intended to be disabled, to functionally assert that they are disabled. Reworked to test all sensors assuming enabled, test disabled intention using the separate function.